### PR TITLE
Restaurants

### PR DESCRIPTION
**Pull request by tonianner**
http://localhost:3000/restaurants ==> can show initial manual data inserted
http://localhost:3000/restaurants/id ==> can show individual restaurant profile (some info not yet in db column), @jackywhlee  is working on it.

### CODE REVIEW

File: app/views/restaurants/_comments.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!-- Comment area-->
+<div class="col-xs-12 col-sm-6 col-sm-offset-3 col-md-4 col-md-offset-4 well">
+
+    <!-- not in DB -->
+    <h4>Leave a Comment:</h4>
```
Comment (dshcheung) [inline app/views/restaurants/_comments.html.erb]: check indentations. 2 spaces, not 4 spaces

File: app/views/restaurants/_comments.html.erb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                <span> * </span>
+                <span> * </span>
+                <span> * </span>
+                <span> * </span>
+                <span> * </span>
+            </span>
+        </h4>
+        Wanted to give this place a try because my friends told me to. The ambiance is fine, but and the salmon is good. But the hand roll was old and smelled like fish. I asked for a table for one, with ended up on a date with a cockroach. I won't go back to this place.
+    </div>
+</div>
```
Comment (dshcheung) [inline app/views/restaurants/_comments.html.erb]: add a new line here

File: app/views/restaurants/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+            <p>
+              <small>
+                <%= @restaurant.address %>
+                <span class="pull-right"><%= @restaurant.price_range %></span>
+              </small>
+            </p>
+        </h1>
+    </div>
+</div>
```
Comment (dshcheung) [inline app/views/restaurants/_header.html.erb]: add a new line

File: app/views/restaurants/index.html.erb [hunk 4]
```diff
@@ -1,3 +1,39 @@
...
+        </p>
+
+        <p><%= restaurant.address %>
+          <span class="pull-right"><%= restaurant.price_range %></span>
+        </p>
+      </div>
+      <img src="<%= restaurant.image_url %>" alt="<%= restaurant.name %>">
+    </div>
+  </div>
 <% end %>
```
Comment (dshcheung) [inline app/views/restaurants/index.html.erb:3]: add a new line

File: app/views/restaurants/_comments.html.erb (hunk 1 above)
Comment (tonianner) [inline app/views/restaurants/_comments.html.erb]: Tnx. I think I may need to fix sublime.. it auto indents to 4 when i press tab...

File: app/views/restaurants/_moreInfo.html.erb [hunk 5]
```diff
@@ -0,0 +1,26 @@
+
+<h3>Location</h3>
+
+<ul>
+    <li><%= @restaurant.address %></li>
```
Comment (dshcheung) [inline app/views/restaurants/_moreInfo.html.erb]: check indentations for this file

File: app/views/restaurants/index.html.erb [hunk 6]
```diff
@@ -1,3 +1,39 @@
+<div class="container">
+
+    <div class="row">
+        <div class="col-lg-12 text-center">
+            <h2>Search Results: <span> NAME SEARCHED</span></h2>
```
Comment (dshcheung) [inline app/views/restaurants/index.html.erb]: check indentation in this file

File: app/views/restaurants/index.html.erb [hunk 7]
```diff
@@ -1,3 +1,39 @@
...
+  <div class="col-sm-4 col-lg-4 col-md-4">
+    <div class="thumbnail">
+      <div class="result-details">
+        <h4><%= restaurant.name %></h4>
+        <%= link_to 'Show', restaurant %>
+        <p> District
+          <span class="pull-right">
+            <span> * </span>
+            <span> * </span>
+            <span> * </span>
```
Comment (dshcheung) [inline app/views/restaurants/index.html.erb]: why do you have code that is very similar to `_header.html.erb`? can you extract these code into a partial?

File: app/views/restaurants/index.html.erb [hunk 8]
```diff
@@ -1,3 +1,39 @@
...
+          </span>
+        </p>
+
+        <p><%= restaurant.address %>
+          <span class="pull-right"><%= restaurant.price_range %></span>
+        </p>
+      </div>
+      <img src="<%= restaurant.image_url %>" alt="<%= restaurant.name %>">
```
Comment (dshcheung) [inline app/views/restaurants/index.html.erb]: use this

http://apidock.com/rails/ActionView/Helpers/AssetTagHelper/image_tag

File: app/views/restaurants/show.html.erb [hunk 9]
```diff
@@ -0,0 +1,19 @@
+
+<div class="container">
+
+  <%= render 'restaurants/header' %>
+
+  <div class="row">
+
+      <div class="col-md-8">
+          <img class="img-responsive" src="<%= @restaurant.image_url %>" alt="<%= @restaurant.name %>" style="width:700px;height:400px;">
```
Comment (dshcheung) [inline app/views/restaurants/show.html.erb]: use this

http://apidock.com/rails/ActionView/Helpers/AssetTagHelper/image_tag

**Comment by tonianner** [comment]
ooo... tnx for the comments! will fix it soon..

File: app/views/restaurants/_header.html.erb (hunk 3 above)
Comment (tonianner) [inline app/views/restaurants/_header.html.erb]: @dshcheung can I ask what you mean by new line?

File: app/views/restaurants/_moreInfo.html.erb (hunk 5 above)
Comment (tonianner) [inline app/views/restaurants/_moreInfo.html.erb]: done

File: app/views/restaurants/index.html.erb (hunk 6 above)
Comment (tonianner) [inline app/views/restaurants/index.html.erb]: done

File: app/views/restaurants/_comments.html.erb [hunk 10]
```diff
@@ -0,0 +1,29 @@
+<!-- Comment area-->
```
Comment (f3r) [inline app/views/restaurants/_comments.html.erb]: To add `comments` you should have a separate model, migration, association and controller for comments. I think you better start with just restaurants, then add this later when it's turn comes ;)

File: app/views/restaurants/_moreInfo.html.erb [hunk 11]
```diff
@@ -0,0 +1,25 @@
+<h3>Location</h3>
+
+<ul>
+  <li><%= @restaurant.address %></li>
+</ul>
+
+<h3>Opening Hours</h3>
+<ul>
+  <li>Mon - Fri <span> 0:00 - 0:00 (not in db)</span></li>
```
Comment (f3r) [inline app/views/restaurants/_moreInfo.html.erb]: If you want to add something in the view, you need to add it in the database, and then access it through the model: https://github.com/wdi-hk-9/Project-SushiMeAway/blob/master/db/migrate/20151102101421_create_restaurants.rb

**Pull request closed without merging**